doc/wg/core/notes/core-notes-2021-06-25.md

```diff
@@ -0,0 +1,115 @@
+# Tock Core Notes 2021-06-25
+
+Attending:
+- Alexandru Radovici
+- Arjun Deopujari
+- Brad Campbell
+- Branden Ghena
+- Hudson Ayers
+- Johnathan Van Why
+- Leon Schuermann
+- Pat Pannuto
+- Phil Levis
+- Vadim Sukhomlinov
+- Anthony Quiroga
+
+## Updates
+
+- Pat: I want to introduce Anthony.  He is going to be working over the summer on the CI work that was
+  getting going.  You can expect a PR of how it is going to work later today after I read over it.
+- Leon: We have been getting inquiries on releasing a new "Tock-registers" version.  Unfortunately, the latest
+  Rust nightly build does break the current released version.  We should look into that.  One PR I released has some discussion
+  and we should discuss how to continue from there and we can release a new version accordingly.
+- Johnathan: If they are asking for an updated Tock-registers version, can't we just update Tock to that version?  This might
```

Comment:
I don't know who said this, but it wasn't me.

Reply:
@jrvanwhy  Sorry for that.  I got your voice mixed up with Branden's.

Reply:
I'm taking this as a huge compliment!